services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -1,15 +1,279 @@
 // components
 import { Box, Heading } from "@chakra-ui/react";
+import { Table } from "components";
 // utils
+import { useEffect, useState } from "react";
 import { useStore } from "utils";
-import { ModalDrawerReportPageShape } from "types";
+import { ModalDrawerReportPageShape, AnyObject } from "types";
+import _ from "lodash";
 
 export const ExportedModalDrawerReportSection = ({
   section: { entityType, verbiage },
 }: Props) => {
+  const [overflow, setOverflow] = useState(false);
   const { report } = useStore() ?? {};
   const entities = report?.fieldData?.[entityType];
 
+  const truncateHeader = (text: string) => {
+    return _.truncate(text, { length: 30 });
+  };
+
+  // if Transition Benchmark Header title has an abbrev.just display that
+  const getTableHeaders = () => {
+    let headers = [];
+    const quarterHeader = "Pop. by Quarter";
+    const bodyHeader = entities.map((entity: AnyObject) =>
+      entity.transitionBenchmarks_targetPopulationName_short
+        ? entity.transitionBenchmarks_targetPopulationName_short
+        : truncateHeader(entity.transitionBenchmarks_targetPopulationName)
+    );
+    const totalHeader = "Total by Quarter";
+
+    headers.push(quarterHeader, ...bodyHeader, totalHeader);
+    return headers;
+  };
+
+  // list of quarters to be added to the table (left column)
+  const quarterLabels = [
+    "2023 Q3",
+    "2023 Q4",
+    "2024 Q1",
+    "2024 Q2",
+    "2024 Q3",
+    "2024 Q4",
+    "2025 Q1",
+    "2025 Q2",
+    "2025 Q3",
+    "2025 Q4",
+    "2026 Q1",
+    "2026 Q2",
+    "2026 Q3",
+  ];
```

Comment:
there are thirteen of these now 😅 
will dynamically generating these be handled elsewhere then?

Reply:
Updated